injectors/desktop/app/index.js

```diff
@@ -46,9 +46,9 @@ const getShelterBundle = () =>
   !localBundle
     ? fetchPromise
     : Promise.resolve(
-        fs.readFileSync(path.join(localBundle, "shelter.js"), "utf8") +
-          `\n//# sourceMappingURL=file:////${path.join(localBundle, "shelter.js.map")}`,
-      );
+      fs.readFileSync(path.join(localBundle, "shelter.js"), "utf8") +
+      `\n//# sourceMappingURL=file:///${process.platform !== "win32" ? "/" : ""}${path.join(localBundle, "shelter.js.map")}`,
```

Comment:
if this fixes source maps, I will test if it also applies on other platforms not just windows later - this does seem like a fairly obvious oversight on our part

once again however, out of scope of this PR.

Reply:
yeah that fixes the sourcemap issue